static/css/devhub/new-landing/sections/more-information.less

```diff
@@ -29,6 +29,7 @@
   ul {
     list-style: none;
     padding: 0;
+    width: 90%;
```

Comment:
This feels like a bit of hack, shouldn't the uls have a specific gutter (via either a padding or margin) between them?